lua/lexers/text.lua

```diff
@@ -1,15 +1,13 @@
--- Copyright 2006-2017 Mitchell mitchell.att.foicica.com. See LICENSE.
+-- Copyright 2006-2020 Mitchell. See LICENSE.
 -- Text LPeg lexer.
 
-local l = require('lexer')
+local lexer = require('lexer')
+local token, word_match = lexer.token, lexer.word_match
+local P, S = lpeg.P, lpeg.S
```

Comment:
Please remove all the unnecessary code here.

Reply:
1) I stuck to the default part of the template, but will take a look at what I can drop from it.

2) I thought I had rebased on top of your scintillua branch, but clearly need to 'fight' a bit more with git.

Reply:
I've made the changes requested and also split the push in two parts, which (after a long wait) indeed shows that modifying the text lexer to the new format is what triggers the build failures.

I can retry without the white space patch, but that will probably make no difference?